packages/functionals/botpress-qna/src/index.js

```diff
@@ -125,31 +125,50 @@ module.exports = {
 
     router.post('/', async (req, res) => {
       try {
+        bp.events.emit('toast.qna-save', { text: 'QnA In Progress', type: 'info', time: 120000 })
+
         const id = await storage.saveQuestion(req.body)
         res.send(id)
+
+        bp.events.emit('toast.qna-save', { text: 'QnA Success', type: 'success' })
       } catch (e) {
         logger.error('QnA Error', e, e.stack)
         res.status(500).send(e.message || 'Error')
+
+        bp.events.emit('toast.qna-save', { text: 'QnA Error', type: 'error' })
```

Comment:
Let's add the error message?

Reply:
@dmytropanontko , I think it makes sense to show it. You can strip it and only show `n` first symbols to make sure it fits the design.